migration/src/m20230608_071249_init_db.rs

```diff
@@ -83,6 +83,7 @@ impl MigrationTrait for Migration {
                     .col(big_unsigned(BatchTransfer::CreatedAt))
                     .col(big_unsigned(BatchTransfer::UpdatedAt))
                     .col(big_unsigned_null(BatchTransfer::Expiration))
+                    .col(boolean(BatchTransfer::ExactExpiry))
```

Comment:
```suggestion
                    .col(boolean_null(BatchTransfer::ExactExpiry))
```
this should be nullable. 

please read carefully the comment I pointed out (https://github.com/RGB-Tools/rgb-lib/pull/38#issuecomment-1843346385), there I said
> as you see the new parameter is nullable